docker/Dockerfile_pretrained_embeddings_spacy_de

```diff
@@ -0,0 +1,34 @@
+FROM python:3.6-slim
+
+ENV RASA_NLU_DOCKER="YES" \
+    RASA_NLU_HOME=/app \
+    RASA_NLU_PYTHON_PACKAGES=/usr/local/lib/python3.6/dist-packages
+
+# Run updates, install basics and cleanup
+# - build-essential: Compile specific dependencies
+# - git-core: Checkout git repos
+RUN apt-get update -qq \
+    && apt-get install -y --no-install-recommends build-essential git-core openssl libssl-dev libffi6 libffi-dev curl  \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+WORKDIR ${RASA_NLU_HOME}
+
+COPY . ${RASA_NLU_HOME}
+
+# use bash always
+RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+
+RUN pip install -r alt_requirements/requirements_pretrained_embeddings_spacy.txt
+
+RUN pip install -e .
+
+RUN pip install https://github.com/explosion/spacy-models/releases/download/de_core_news_sm-2.0.0/de_core_news_sm-2.0.0.tar.gz --no-cache-dir > /dev/null \
+    && python -m spacy link de_core_news_sm de
+
+VOLUME ["/app/projects", "/app/logs", "/app/data"]
+
+EXPOSE 5000
+
+ENTRYPOINT ["./entrypoint.sh"]
```

Comment:
Both images differ regarding the language model. However, I did not find a clean way to reuse the images without having a certain order of docker builds. But could we maybe add the model download / link to the entrypoint so that users can specify it with environment variables?

Reply:
by 'reuse' you mean to use one as a base at build time? I think we can optimize for that later. I think the dockerfiles should be 'batteries included' so that users don't have to first run an entrypoint command to download & install the language model (which can take a long time). lmk if I misunderstood

Reply:
> by 'reuse' you mean to use one as a base at build time?

Yes, exactly. But as far as I've seen this is only possible if you build a base image first, and then extend this image. 

> I think the dockerfiles should be 'batteries included'
Makes sense :+1: